ui/app/AppLayouts/Profile/views/profile/MyProfileSettingsView.qml

```diff
@@ -0,0 +1,152 @@
+                               || biometricsSwitch.checked != biometricsSwitch.currentStoredValue
+
+    readonly property bool valid: !!displayNameInput.text && displayNameInput.valid
+
+    function reset() {
+        displayNameInput.text = Qt.binding(() => { return profileStore.displayName })
+        biometricsSwitch.checked = Qt.binding(() => { return biometricsSwitch.currentStoredValue })
+    }
+
+    function save() {
+        profileStore.setDisplayName(displayNameInput.text)
+
+        if (biometricsSwitch.checked)
+            Global.openPopup(storePasswordModal)
+        else
+            localAccountSettings.storeToKeychainValue = Constants.storeToKeychainValueNever;
+    }
+
+    function offerToStorePassword(password, runStoreToKeyChainPopup)
+    {
+        if (Qt.platform.os !== "osx")
+            return;
+
+        localAccountSettings.storeToKeychainValue = Constants.storeToKeychainValueStore;
+        root.privacyStore.storeToKeyChain(password);
+    }
+
+    ProfileHeader {
+        Layout.fillWidth: true
+        Layout.leftMargin: Style.current.padding
+        Layout.rightMargin: Style.current.padding
+
+        displayName: profileStore.name
+        pubkey: profileStore.pubkey
+        icon: profileStore.icon
+        imageSize: ProfileHeader.ImageSize.Big
+        
+        displayNameVisible: false
+        pubkeyVisible: false
+        emojiHashVisible: false
+        editImageButtonVisible: true
+    }
+
+    StatusInput {
+        id: displayNameInput
+        Layout.fillWidth: true
+        label: qsTr("Display name")
+        input.placeholderText: qsTr("Display Name")
+        charLimit: 24
+        input.text: root.profileStore.displayName
+        validators: Constants.validators.displayName
+    }
+
+    StatusListItem {
+        Layout.fillWidth: true
+        visible: Qt.platform.os == "osx"
+        title: qsTr("Biometric login and transaction authentication")
+        icon.name: "touch-id"
+        components: [ StatusSwitch {
+            id: biometricsSwitch
+            readonly property bool currentStoredValue: localAccountSettings.storeToKeychainValue === Constants.storeToKeychainValueStore
+            checked: currentStoredValue
+        } ]
+        sensor.onClicked: biometricsSwitch.toggle()
+    }
+
+    StatusTabBar {
+        id: showcaseTabBar
+        Layout.fillWidth: true
+
+        StatusTabButton {
+            width: implicitWidth
+            text: qsTr("Communities")
+        }
+        StatusTabButton {
+            width: implicitWidth
+            text: qsTr("Accounts")
+        }
+    }
+
+    StackLayout {
+        Layout.fillWidth: true
+        currentIndex: showcaseTabBar.currentIndex
+
+        Column {
+            Repeater {
+                model: communitiesModule.model
+
+                CommunityDelegate {
+                    Layout.fillWidth: true
+                    visible: joined
```

Comment:
@igor-sirotin if we want to display only joined communities why don't we have appropriate model for that? This way we have `CommunityDelegate` instantiated for each community regardless it's joined or not.